### ISSUE TITLE
Configuration of E2E encryption in a room

### ISSUE THREAD

**Issue by benparsons** [matrix-org/matrix-doc#1200]
Documentation: https://docs.google.com/document/d/1SEPMhNh6ztcrrbkGRSayVQ23bd3cfMPkTgGL4kBS9Ps/edit#heading=h.e7hfigo2zcsj
Author: @richvdh
Date: 16/06/2016
PRs: #1284

**Comment by richvdh** [comment]
part of https://github.com/matrix-org/matrix-doc/issues/501. I don't think it has been merged.

**Comment by Zil0** [comment]
Isn't this a duplicate of #590?

**Comment by turt2live** [comment]
This is the proposal that fixes #590

**Comment by Zil0** [comment]
@turt2live right, so shouldn't #590 be closed?

**Comment by turt2live** [comment]
Once the spec PR for this proposal is merged, yes

**Comment by richvdh** [comment]
@Zil0 the convention we've gone with is that each proposal doc gets its own issue, even if there is already a related issue open about it. It's leading to a bit of duplication but such is life.

As far as this particular pair of issues is concerned: I'm not actually sure that this proposal does fix all of #590. There's quite a shopping list of things over there, and I'm not sure that it's fair to say this is a duplicate.

**Comment by Zil0** [comment]
@richvdh I thought it was a duplicate because I didn't know about the convention, I get it now.

Looking closer at the shopping list, you're right that its scope seems larger than this proposal, but it feels very outdated overall, and it is not obvious what would be left to fix in order to close it.

**Comment by uhoreg** [comment]
Fixed by #1284
